working_with_fhir_connector/README.md

```diff
@@ -0,0 +1,11 @@
+# FHIR Connector Sample
+This sample demonstrate how to use pre-built FHIR connector to connect to 
```

Comment:
This sample demonstrate**s** how to use **the** pre-built FHIR connector to connect to Cerner EMR.